packages/api-client/src/api/serializers/product.ts

```diff
@@ -145,10 +161,11 @@ const partialDeserializeProductVariant = (
     original: variant.attributes.price,
     current: variant.attributes.price
   },
-  inStock: variant.attributes.in_stock
+  inStock: variant.attributes.in_stock,
+  vendor: vendoMarketplace ? deserializeVendor(attachments, product) : null
```

Comment:
Do we need that vendoMarketplace parameter here? We could just based it on whether there is a vendor reference in a product and whether vendor is found in attachments